src/it/resources/com/google/checkstyle/test/chapter4formatting/rule42blockindentation/ClassWithChainedMethods.java

```diff
@@ -0,0 +1,39 @@
+package com.google.checkstyle.test.chapter4formatting.rule42blockindentation; // indent:0 exp:0
+
+public class ClassWithChainedMethods { // indent:0 exp:0
+
+  public ClassWithChainedMethods(Object... params) { // indent:2 exp:2
+  } // indent:2 exp:2
+
+  public static void main(String[] args) { // indent:2 exp:2
+    new ClassWithChainedMethods()
+        .getInstance("string_one") // indent:4 exp:4
+        // violation below ''method call' child has incorrect indentation level 4, expected .* 8.'
+    .doNothing("string_one".trim(), "string_two"); // indent:15 exp:>=8
+
+    int length =
+        new ClassWithChainedMethods(
+                "param1", // indent:4 exp:4
+                "param2")
+            .getInstance() // indent:32 exp:>=8
+      // violation below ''.' has incorrect indentation level 6, expected level should be 8.'
+      .doNothing("nothing")
+      .length();
+      // violation above ''.' has incorrect indentation level 6, expected level should be 8.'
+
+    int length2 = // indent:4 exp:4
+        // violation below ''new' has incorrect indentation level 4, expected level should be 8.'
```

Comment:
this file already contained violations, so we can ignore this.

Formatter corrected these indentations but I reverted them to as they were before.

Reply:
please put violation comment to be above `int length2 =`
comment in between of `=` and statement is weird edge case, lets not deal with it there.